broadlink/alarm.py

```diff
@@ -27,30 +27,22 @@ def get_sensors_status(self) -> dict:
         if not payload:
             return None
         count = payload[0x4]
-        sensors = payload[0x6:]
```

Comment:
I think we should rename this method to check_sensors() in order to create a common interface for all sensors. Another idea is to use the serials as keys in the returned dictionary so we can have them as unique ids and easily coordinate updates in Home Assistant (just like we do for mp1 slots).

Reply:
I will leave to do this at the right time, there is no reason for us to break it now. I am just making my intentions clear.